src/java-runtime/java/mono/android/MonoPackageManager.java

```diff
@@ -42,6 +45,14 @@ public static void LoadApplication (Context context, ApplicationInfo runtimePack
 				String dataDir      = getNativeLibraryPath (context);
 				ClassLoader loader  = context.getClassLoader ();
 				String runtimeDir = getNativeLibraryPath (runtimePackage);
+				int localDateTimeOffset;
+
+				if (android.os.Build.VERSION.SDK_INT >= android.os.Build.VERSION_CODES.O) {
+					localDateTimeOffset = OffsetDateTime.now().getOffset().getTotalSeconds();
```

Comment:
Have we tested this on an API-21 device or emulator?  We *may* need to move this expression into a separate method; I am not sure.

Reply:
Just tried it on API 21 emulator
![Screen Shot 2022-09-14 at 1 42 58 PM](https://user-images.githubusercontent.com/16830051/190225413-2aaa88c6-a799-4931-aa73-b1cd4a7803b2.png)
, I dont have an API 21 device. It looks like it works fine? Unless I did something wrong, why might it not work on API-21 device/emulator if we have the runtime API check? How would moving it to a separate method remedy this?